core/templates/dev/head/base.js

```diff
@@ -211,6 +213,17 @@ oppia.controller('Base', [
       return false;
     };
 
+    //  Method to check if the window size is narrow
+    $scope.isWindowNarrow = function() {
```

Comment:
Actually, I suspect this is going to work fine even if you don't register the hook, thanks to Angular's two-way binding. Is that true? I think the hook is only necessary if you pull windowWidth out into an integer variable and recalculate it on each resize.

If I'm wrong, please ignore this comment. Otherwise, you might want to do exactly that (i.e. pull it out into a variable) to avoid Angular automatically making unnecessary calls to $scope.isWindowNarrow().